### PR TITLE
Fix issue with displaying SCIM2 "supported by default" claims in user profile

### PR DESCRIPTION
**Pull request by NipuniBhagya**

### CODE REVIEW

File: apps/console/src/features/users/components/user-profile.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@
  * under the License
  */
 
+import { ProfileConstants } from "@wso2is/core/constants"
```
Comment (DimalChandrasiri) [inline apps/console/src/features/users/components/user-profile.tsx:19]: Missing `;`

File: apps/console/src/features/users/components/user-profile.tsx [hunk 2]
```diff
@@ -207,9 +214,17 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
...
-                            ? { emails: [values.get(schema.name)] }
-                            : { [schemaNames[0]]: values.get(schemaNames[0]) };
+                        if (schema.extended) {
+                            opValue = {
+                                [ProfileConstants.SCIM2_ENT_USER_SCHEMA]: {
+                                    [schemaNames[0]]: values.get(schemaNames[0])
+                                }
+                            }
+                        } else {
+                            opValue = schemaNames[0] === "emails"
```
Comment (DimalChandrasiri) [inline apps/console/src/features/users/components/user-profile.tsx:224]: Shouldn't the "emails" term moved to a constant?

File: apps/myaccount/src/api/profile.ts [hunk 3]
```diff
@@ -17,13 +17,14 @@
  */
 
 import { IdentityClient } from "@asgardio/oidc-js";
+import { ProfileConstants } from "@wso2is/core/constants"
```
Comment (DimalChandrasiri) [inline apps/myaccount/src/api/profile.ts:20]: Missing `;`

File: apps/myaccount/src/components/profile/profile.tsx [hunk 4]
```diff
@@ -17,6 +17,7 @@
  */
 
 import { updateProfileImageURL } from "@wso2is/core/api";
+import { ProfileConstants } from "@wso2is/core/constants"
```
Comment (DimalChandrasiri) [inline apps/myaccount/src/components/profile/profile.tsx:20]: Missing `;`

**Review by thivi — APPROVED** [review]

File: apps/console/src/features/users/components/user-profile.tsx (hunk 1 above)
Comment (NipuniBhagya) [reply apps/console/src/features/users/components/user-profile.tsx]: Fixed with d523fb9

File: apps/console/src/features/users/components/user-profile.tsx (hunk 2 above)
Comment (NipuniBhagya) [reply apps/console/src/features/users/components/user-profile.tsx]: Fixed with d523fb9

File: apps/myaccount/src/api/profile.ts (hunk 3 above)
Comment (NipuniBhagya) [reply apps/myaccount/src/api/profile.ts]: Fixed with d523fb9

File: apps/myaccount/src/components/profile/profile.tsx (hunk 4 above)
Comment (NipuniBhagya) [reply apps/myaccount/src/components/profile/profile.tsx]: Fixed with d523fb9

**Review by DimalChandrasiri — APPROVED** [review]

**Pull request merged**